CONTRIBUTING.md

```diff
@@ -0,0 +1,85 @@
+
+Any XRPL-Standards document can be referred to interchangeably as an "XLS", "XRPL Standard", or "document".
+
+## Summary
+
+Copyright on all content is subject to the terms of the [license](LICENSE).
+
+All contributors grant everyone:
+
+Copyright: a royalty-free license to use the copyrights for their contributions.
+Patent: a commitment to license on a royalty-free basis their essential patent claims reading on their contributions.
+
+## Background
+
+The work of the XRP Ledger community is open source, collaborative, and welcoming of all contributors participating in good faith. [Learn more about the XRP Ledger at XRPL.org](https://xrpl.org/).
+
+
+## Process
+
+The XRPL-Standards process attempts to be easy to use, but also rigorous enough that there are permalinks to revisions of documents for reference purposes.
+
+### Gathering Feedback Before Submitting
+
+Please gather community input before opening a PR. Collecting such feedback helps to refine your concept. This step is required.
+
+Start a [Discussion](https://github.com/XRPLF/XRPL-Standards/discussions) under this repo.
+
+The title should follow the naming convention `XLS-0000d {Title}`, where `0000` is a unique number for the XLS, `d` indicates that the document is a Draft (in progress), and `{Title}` is a descriptive title for the proposed document.
+
+Use the next number that has not yet been used. If a conflict occurs, it will be fixed by a maintainer or editor. Maintainers or editors also reserve the right to remove or re-number proposals as necessary.
+
+The number is important, as it will be used to reference features and ideas throughout the community.
+
+Discussions are suitable for early work-in-progress: ask, suggest, add, and make sweeping changes.
+
+Next, the discussion should be converted to a **PULL REQUEST** which adds the standard, as a Markdown file, to the repo with one or more commits.
+
+The content of the discussion should be edited into a summary, with a link to the PR.
+
+The last comment on the discussion should also be a link to the PR.
+
+Finally, the discussion should be closed from further comments, with further comments instead being posted on the PR for fine-tuning and alignment with implementation or adoption (as appropriate).
+
+When opening a PR, there are two document types: *Drafts* and *Candidate Specifications*. The type and status of any particular document has no bearing on the priority of that document. Typically, the further along in the process, the more likely it is that any particular XLS will be implemented or adopted.
+
+### Drafts
+
+A _Draft_ is a document that proposes some new feature, protocol or idea related to the XRP Ledger. The criteria for getting the document merged is minimal: project maintainers will review the document to ensure style conformance and applicability, but will otherwise not require editorial fixes before allowing it to be published.
+
```

Comment:
```suggestion
A _Draft_ is a document that proposes some new feature, protocol or idea related to the XRP Ledger. The criteria for getting the document merged is minimal: project maintainers will review the document to ensure style conformance and applicability, but will otherwise not require editorial fixes before allowing it to be published.

A document does not need to have an implementation in order to become a Draft. A Draft may or may not have implementation(s) available; no code is required prior to the Draft being published.
```